feat: allow adding extra ports to the application

omBratteng opened the pull request:
Adds the possibility to add more ports to the application service

idoshamun reviewed (commented):
@omBratteng don't we need to add extra ports to the container as well?

omBratteng commented:
Yes, probably 🙈

idoshamun commented:
@omBratteng also I'd suggest a different approach. Allow providing the ports array and by default set it to port 80 like we do now

omBratteng commented:
Good idea. Will change it 😄

omBratteng commented:
@idoshamun marked the `port` as deprecated and added new `ports` which takes array of ports instead. Renamed `extraServicePorts` to `servicePorts` and default back to port 80 if no ports are given

idoshamun commented on src/k8s.ts:
```diff
@@ -379,6 +382,12 @@ export const createAutoscaledExposedApplication = ({
   }
+
+  const ports =
+    servicePorts && servicePorts.length > 0
```
```suggestion
    servicePorts?.length > 0
```

idoshamun reviewed (approved):
Perfect, please announce on #engineering once we release a new version

idoshamun commented:
@omBratteng I see build fails, please fix it before merging

omBratteng commented:
The build failed with your suggestion 🙈 But it should work... maybe just set default `servicePorts = []`?
